### PR TITLE
Add pipelines for bazel-lib and rules_license to TF config

### PR DESCRIPTION
**Pull request by fweikert**
Progress towards #1309, #1313

### CODE REVIEW

File: buildkite/terraform/bazel/main.tf [hunk 1]
```diff
@@ -1822,3 +1822,35 @@ resource "buildkite_pipeline" "bazel-bazel" {
...
+  team = [{ access_level = "BUILD_AND_READ", slug = "bazel" }]
+  provider_settings {
+    trigger_mode = "code"
+    build_pull_requests = true
+    skip_pull_request_builds_for_existing_commits = true
+    build_pull_request_forks = true
+    prefix_pull_request_fork_branch_names = true
+    publish_commit_status = true
+  }
+}
```
Comment (meteorcloudy) [inline buildkite/terraform/bazel/main.tf:1856]: nit: new line

**Review by meteorcloudy — APPROVED** [review]
Should we also add them in downstream?

**Comment by meteorcloudy** [comment]
Is this still needed?

**Comment by aiuto** [comment]
Could we get this PR going again.  I am trying to set up the CI runs for rules_license, but my tests are failing completely in a way I don't understand.
https://buildkite.com/bazel/rules-license/builds/3#01826f31-5cb1-40d6-8b54-e35eabf585f0

The error is.
```
2022-08-05 18:08:14 FATAL  Config file is empty

  | 🚨 Error: The command exited with status 1
```
which I don't understand at all, nor can I find the message in the code here, so I don't know where to begin.

**Comment by aiuto** [comment]
Hi. Friendly ping.
I need to get CI running for rules_license now that bazel depends on it.

**Comment by fweikert** [comment]
This PR is a no-op since Philipp already added them in https://github.com/bazelbuild/continuous-integration/commit/7bf04992182577a3d7aefea249c1aa23dd8e00e1

**Comment by fweikert** [comment]
Re https://buildkite.com/bazel/rules-license/builds/3#01826f31-5cb1-40d6-8b54-e35eabf585f0: your yaml config is invalid

**Pull request closed without merging**